calamus/fields.py

```diff
@@ -241,6 +242,7 @@ def schema(self):
                         context=context,
                         load_only=self._nested_normalized_option("load_only"),
                         dump_only=self._nested_normalized_option("dump_only"),
+                        lazy=self.parent.lazy,
```

Comment:
We can use `self.root` to get the parent schema. It would work for `List`'s elements too.